cypress/integration/participantStatus.spec.js

```diff
@@ -9,21 +29,26 @@ describe('Participants status suite', () => {
     cy.contains('button', 'View My Candidates').click();
   };
 
-  const interviewParticipantForm = (siteId) => {
+  const interviewParticipantForm = () => {
     // let it select today as the date
-    cy.get('input[name=ContactedDate]').clear().type('2022/01/13');
+    // Ensure dates selected are within the phase range of Sacred Macaw
+    cy.get('input[name=ContactedDate]').clear().type('2023/01/13');
     cy.contains('button', 'Submit').click();
   };
 
   const hireParticipantForm = (siteId) => {
     // both dates
-    cy.get('input[name=DateHired]').clear().type('2022/01/14');
-    cy.get('input[name=StartDate]').clear().type('2022/01/15');
+    // Ensure dates selected are within the phase range of Sacred Macaw
+    cy.get('input[name=DateHired]').clear().type('2023/01/14');
+    cy.get('input[name=StartDate]').clear().type('2023/01/15');
     // select site
     cy.get('#mui-component-select-site').click();
     cy.get(`li[data-value='${siteId}']`).click();
     // acknowledge participant accepted offer in writing
     cy.get('input[name="acknowledge"]').click();
+    //  expect alert with allocations/remainingHires to exist
+    cy.wait(1000);
```

Comment:
Why the one second wait? I'd like to see at least a comment explaining why - although ideally I'd like to think there's a better solution here, especially considering how long our tests already take. Anyone with more Cypress experience (@fw-noel?) want to chime in here?

Reply:
I am curious if its necessary. Its probably to wait for the alert, but are our other tests needing it?

Reply:
It seems to be used in a few other tests, so with consistency in mind, I'm okay with this. Wonder if they can be shortened to a half second or something

Reply:
So - the test was passing locally with a `wait(1000)`, but not in the pipeline. I increased the wait to `wait(2000)` and the tests are now passing in the pipeline. The wait is necessary to allow the component to complete the async functions before the alert exists on the UI.